### PR TITLE
Add load transaction functionality

### PR DESCRIPTION
**Pull request by Mrtenz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -117,7 +117,7 @@
     "@ethersproject/bytes": "5.1.0",
     "@ethersproject/json-wallets": "5.1.0",
     "@ethersproject/random": "5.1.0",
-    "@ethersproject/transactions": "5.1.0",
+    "@ethersproject/transactions": "5.2.0",
```
Comment (FrederikBolding) [inline package.json:120]: Can we bump them all since `@ethersproject/transactions` needs them all anyway?

**Comment by Mrtenz** [comment]
[ch8588]

File: src/types/jsonrpc.ts [hunk 2]
```diff
@@ -29,7 +29,18 @@ export const SignTransactionStruct = tuple([
   })
 ]);
 
-export type TSignTransaction = Infer<typeof SignTransactionStruct>;
```
Comment (FrederikBolding) [inline src/types/jsonrpc.ts:32]: Why?

File: src/app/screens/LoadTransaction.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+const SCHEMA = object({
+  account: string()
+    .required(translateRaw('ACCOUNT_EMPTY'))
+    .typeError(translateRaw('ACCOUNT_EMPTY')),
+  transaction: string()
+    .required(translateRaw('TRANSACTION_EMPTY'))
+    .test('valid-raw-transaction', translateRaw('NOT_RAW_TRANSACTION'), (value) => {
+      try {
+        const transaction = parse(value);
+        return bigify(transaction.r ?? 0).isZero() && bigify(transaction.s ?? 0).isZero();
```
Comment (FrederikBolding) [inline src/app/screens/LoadTransaction.tsx:27]: Can we move this to utility function?

File: src/app/screens/LoadTransaction.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      }),
+      true
+    );
+
+    dispatch(enqueue(queueTransaction));
+    dispatch(push(ROUTE_PATHS.HOME));
+  };
+
+  const handleSelectAccount = (account: IAccount) => {
+    form.setValue('account', account.uuid);
```
Comment (FrederikBolding) [inline src/app/screens/LoadTransaction.tsx:72]: Can we do this as a oneliner?

File: src/app/components/AccountSelector.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      label={data.label}
+      truncate={false}
+      bg="none"
+      onClick={handleSelect}
+    />
+  );
+};
+
+const AccountValue = ({ value }: { value: IAccount }) => (
+  <Account address={value.address} label={value.label} truncate={false} bg="none" />
```
Comment (FrederikBolding) [inline src/app/components/AccountSelector.tsx:28]: Has padding issues

File: src/app/components/TxDetails.tsx [hunk 6]
```diff
@@ -7,19 +7,32 @@ import { bigify } from '@common/utils';
...
+}) => {
   const chain = getChain(tx.chainId);
   const maxTxFee = bigify(tx.gasPrice).multipliedBy(bigify(tx.gasLimit));
   const symbol = chain?.nativeCurrency?.symbol ?? '?';
   const network = chain?.name ?? translateRaw('UNKNOWN_NETWORK');
   const data = tx.data?.toString() ?? '0x';
+
   return (
     <>
+      {/* @todo Update design */}
```
Comment (FrederikBolding) [inline src/app/components/TxDetails.tsx:28]: Remember this

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/types/jsonrpc.ts (hunk 2 above)
Comment (Mrtenz) [reply src/types/jsonrpc.ts:32]: The type for `to` is inferred as `string`, but it can also be undefined, causing type errors elsewhere.

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**